### ISSUE TITLE
Where to find it icon lacks descriptive text

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1062]
Level AAA violation: Links must have discernible text.

`.library-location > .call-number > a.find-it` has no text. Add `aria-label="Where to find it"` to the `a` tag in [availability.js](https://github.com/pulibrary/orangelight/blob/a0c52d35af5d0f2014787202eb5e0102efc9d753/app/assets/javascripts/availability.js.coffee.erb#L260)
